web-common/src/metrics/service/MetricsService.ts

```diff
@@ -52,7 +52,7 @@ export class MetricsService
   public async loadCommonFields() {
     const projectPathParts = this.localConfig.project_path.split("/");
     this.commonFields = {
-      app_name: "rill-developer",
```

Comment:
Same note, this might not be safe to change. And shouldn't matter keeping it at the old value.